### PR TITLE
Added test key.

### PR DESCRIPTION
**Pull request by michaelgsharp**
Added back in the test key to fix the extension issue.

### CODE REVIEW

**Comment by eerhardt** [comment]
Does CI run on this branch? I don't see any checks.

File: src/Microsoft.ML.Core/ComponentModel/ComponentCatalog.cs [hunk 1]
```diff
@@ -1110,12 +1110,7 @@ private void LoadExtensions(Assembly assembly, bool throwOnError)
         /// </remarks>
         private static bool CanContainExtensions(Assembly assembly)
         {
-            // We used to have two separate strong name keys for product and test binaries and Arcade
-            // does not support that anymore (and that did not accept our PR for adding the additional
-            // signed key and using two strong name keys without the PR). If they ever decide to support
-            // this scenario we can remove the regex and go back to checking just the public key and startswith.
             if (assembly.FullName.StartsWith("Microsoft.ML.", StringComparison.Ordinal)
-               && !Regex.IsMatch(assembly.FullName, @"Microsoft\.ML.*\.Tests.*")
                && HasMLNetPublicKey(assembly))
```
Comment (eerhardt) [inline src/Microsoft.ML.Core/ComponentModel/ComponentCatalog.cs:1114]: (nit) your previous change had some whitespace issues going from 4 spaces to 3 on this line, and on the `return false;` line below. Can you fix those while you are in here?

File: src/Microsoft.ML.Core/PublicKey.cs [hunk 2]
```diff
@@ -15,6 +15,8 @@ namespace Microsoft.ML.Internal.CpuMath.Core
     internal static class PublicKey
     {
         public const string Value = ", PublicKey=00240000048000009400000006020000002400005253413100040000010001004b86c4cb78549b34bab61a3b1800e23bfeb5b3ec390074041536a7e3cbd97f5f04cf0f857155a8928eaa29ebfd11cfbbad3ba70efea7bda3226c6a8d370a4cd303f714486b6ebc225985a638471e6ef571cc92a4613c00b8fa65d61ccee0cbe5f36330c9a01f4183559f1bef24cc2917c6d913e3a541333a1d05d9bed22b38cb";
+        public const string TestValue = ", PublicKey=002400000480000094000000060200000024000052534131000400000100010015c01ae1f50e8cc09ba9eac9147cf8fd9fce2cfe9f8dce4f7301c4132ca9fb50ce8cbf1df4dc18dd4d210e4345c744ecb3365ed327efdbc52603faa5e21daa11234c8c4a73e51f03bf192544581ebe107adee3a34928e39d04e524a9ce729d5090bfd7dad9d10c722c0def9ccc08ff0a03790e48bcd1f9b6c476063e1966a1c4";
+
```
Comment (eerhardt) [inline src/Microsoft.ML.Core/PublicKey.cs:19]: (nit) unnecessary newline.

**Review by eerhardt — APPROVED** [review]
Looks good.

**Comment by michaelgsharp** [comment]
@eerhardt it didn't run against the CI because this is just going into our arcade branch for now. As soon as we get arcade fully working we will make a PR from that branch into master. We are doing the PR's here just so that we can get feedback and have more visibility.

File: src/Microsoft.ML.Core/ComponentModel/ComponentCatalog.cs (hunk 1 above)
Comment (michaelgsharp) [reply src/Microsoft.ML.Core/ComponentModel/ComponentCatalog.cs]: Not sure how that happened but its fixed now.

File: src/Microsoft.ML.Core/PublicKey.cs (hunk 2 above)
Comment (michaelgsharp) [reply src/Microsoft.ML.Core/PublicKey.cs]: Removed.

**Pull request merged**